gems/sorbet/lib/fetch-rbis.rb

```diff
@@ -29,7 +29,17 @@ class Sorbet::Private::FetchRBIs
   # Ensure our cache is up-to-date
   T::Sig::WithoutRuntime.sig {void}
   def self.fetch_sorbet_typed
-    if !File.directory?(RBI_CACHE_DIR)
+    if File.directory?(RBI_CACHE_DIR)
+      cached_remote = IO.popen(["git", "-C", RBI_CACHE_DIR, "config", "--get", "remote.origin.url"]) {|pipe| pipe.read}.strip
+
+      # Compare the <owner>/<repo>.git to be agnostic of https vs ssh urls
+      cached_remote_repo = cached_remote.split(%r{github.com[:/]}).last
+      requested_remote_repo = SORBET_TYPED_REPO.split(%r{github.com[:/]}).last
```

Comment:
This makes the assumption that forks would only ever be hosted on github. I think this is probably fine for now, but it would make a good enhancement issue :)

Reply:
Yea I went back and forth on that. My initial attempt was a regex but I felt this was simpler and has a nice property of gracefully falling back simply checking the whole string, just not https/ssh agnostic:
```ruby
> 'git@gitlab:x/y.git'.split(%r{github.com[:/]}).last
=> "git@gitlab:x/y.git"
```